src/plugins/telemetry/public/services/telemetry_sender.test.ts

```diff
@@ -66,6 +66,22 @@ describe('TelemetrySender', () => {
   });
 
   describe('shouldSendReport', () => {
+    beforeEach(() => {
+      document.hasFocus = jest.fn().mockReturnValue(true);
+    });
```

Comment:
NIT: for global/existing objects mocking, `jest.spyOn` is usually recommended rather than aggressively replacing the property.


```ts
let hasFocus: jest.Spy;

beforeEach(() => {
   hasFocus = jest.spyOn(document, 'hasFocus');
});

afterEach(() => {
   hasFocus.mockRestore();
});


// ... in test

hasFocus.mockReturnValue(false);